src/server/index.tsx

```diff
@@ -48,7 +48,18 @@ server.get("/*", async (req, res) => {
   };
 
   // Get site data first
-  let site = await initialFetchReq.client.getSite(getSiteForm);
+  // This bypasses errors, so that the client can hit the error on its own,
+  // in order to remove the jwt on the browser. Necessary for wrong jwts
+  let try_site: any = await initialFetchReq.client.getSite(getSiteForm);
+  if (try_site.error == "not_logged_in") {
+    console.error(
+      "Incorrect JWT token, skipping auth so frontend can remove jwt cookie"
+    );
+    delete getSiteForm.auth;
+    delete initialFetchReq.auth;
+    try_site = await initialFetchReq.client.getSite(getSiteForm);
+  }
+  let site: GetSiteResponse = try_site;
```

Comment:
Ah so this is to handle the invalidated jwts from the backend change :+1:

Reply:
Yes... I ran into this while using incorrect JWT tokens on the federation test instances. Its necessary because the server can't reset your cookies, only the browser can, and this at least lets the server-side do an initial user-less fetch without errors, so that the front-end can run into one, and reset the cookies.

Reply:
Might be worth mentioning this in the API docs.